src/saving/Save.cs

```diff
@@ -86,6 +86,7 @@ public static SaveInformation LoadJustInfoFromSave(string saveName)
         catch (Exception e)
         {
             GD.PrintErr($"Failed to load save info from ${saveName}, error: ${e}");
+            LogErrorToFile(saveName, "BROKEN", e);
```

Comment:
This isn't the right place to log this as this gets called when the save list is opened.